mda/rootfs/etc/dovecot/conf.d/90-fts-xapian.conf.templ

```diff
@@ -1,26 +1,28 @@
 {{ $enable_fts := eq (or ($.Env.ENABLE_FTS) "") "true" }}
 {{ $fts_args := default .Env.FTS_ARGS "partial=3 full=20 verbose=0" }}
 
-
 {{ if $enable_fts }}
+mail_plugins = $mail_plugins fts fts_xapian
```

Comment:
Does it definitely work when defined here? During my testing I found I had to put it in the main `10-mail.conf` because it wouldn't work for me in the `90-fts...` config. I might have done it wrong, but I thought it was because of when/how the plugin list gets defined.